shunting-yard-binary/src/main/resources/shunting-yard-minimal.yml.template

```diff
@@ -5,7 +5,7 @@ replica-catalog:
   name: target-data-warehouse
   hive-metastore-uris: thrift://domain-name.eu-west-1.compute.amazonaws.com:9083
 event-receiver:
-  message-reader-factory-class: com.hotels.shunting.yard.receiver.sqs.messaging.SqsMessageReaderFactory
+  message-reader-factory-class: com.expediagroup.shuntingyard.receiver.sqs.messaging.SqsMessageReaderFactory
   configuration-properties:
     com.hotels.shunting.yard.event.receiver.sqs.queue: https://sqs.us-west-2.amazonaws.com/1234567/sqs-queue
```

Comment:
What are we going to do with these configuration properties that have `com.hotels` in them? Maybe we should just remove the `com.hotels` from all of them?

Reply:
Removed the package name from the config properties.